webhooks-extension/tekton/publish.yaml

```diff
@@ -61,21 +68,41 @@ spec:
       cd /workspace/go/src/github.com/tektoncd/experimental/webhooks-extension
 
       # Publish images and create release.yaml
-      ko resolve --preserve-import-paths -f /workspace/go/src/github.com/tektoncd/experimental/webhooks-extension/config/ > /workspace/bucket/webhooks-extension/latest/release.yaml
+      rm /workspace/output/bucket/latest/*.yaml
+      ko resolve --preserve-import-paths -f /workspace/go/src/github.com/tektoncd/experimental/webhooks-extension/config/ > /workspace/output/bucket/latest/tekton-webhooks-extension-release.yaml
+      ko resolve --preserve-import-paths -f /workspace/go/src/github.com/tektoncd/experimental/webhooks-extension/config/openshift/openshift-tekton-webhooks-extension-release.yaml > /workspace/output/bucket/latest/openshift-tekton-webhooks-extension-release.yaml
+
     volumeMounts:
       - name: gcp-secret
         mountPath: /secret
 
-  - name: copy-to-tagged-bucket
+  - name: pin-docker-image-refs
+    image: python
+    workingDir: "/workspace/output/bucket"
+    command:
+    - /bin/sh
+    args:
+    - -ce
+    - |
+      curl https://raw.githubusercontent.com/tektoncd/dashboard/master/tekton/scripts/lockdown.py --output lockdown.py
+      chmod +x lockdown.py
+      pip install docker
+      ./lockdown.py --omit webhooks-extension --path latest/tekton-webhooks-extension-release.yaml
+      ./lockdown.py --omit webhooks-extension --path latest/openshift-tekton-webhooks-extension-release.yaml
+    volumeMounts:
```

Comment:
Changes look good now, how come we're now doing docker socket mounting?

Reply:
for the record it's required by the lockdown.py script